docs/user/guides/reproducible-builds.rst

```diff
@@ -252,3 +139,26 @@ and it generates a ``requirements.txt`` file with the full set of transitive dep
           # via sphinx
       urllib3==1.26.13
           # via requests
+
+Conclusion
+-----------
+
+If you followed this guide,
+you have pinned:
+
+* tool versions (Python, Node)
+* top-level dependencies (Sphinx, Sphinx extensions)
+* transitive dependencies (Pytz, Jinja2)
+
+This will ensure your builds don't fail because of a random tool or dependency update.
+You do still need to ensure you're upgrading your dependencies from time to time,
+but you can do it on your own schedule.
+
+.. seealso::
+
+   :doc:`/config-file/v2`
+    Configuration File Reference
```

Comment:
```suggestion
    Configuration file reference
```

Noting for later to add something in our glossary / style guide here, too. I think we should call this "configuration file" but even better add `.readthedocs.yaml`.